packages/babel-plugin-proposal-pipeline-operator/test/fixtures/fsharp/arrow-functions/input.js

```diff
@@ -0,0 +1,14 @@
+var result = [5,10]
+  |> (_ => _.map(x => x * 2))
+  |> (_ => _.reduce( (a,b) => a + b ))
+  |> (sum => sum + 1)
+
+expect(result).toBe(31);
```

Comment:
Nit: to make `input` tests smaller, all the non relevant parts (i.e. line 6, 9, 10 and 14) can be removed.

Reply:
I'm trying to keep exec == input to make reviewing output test failures easier. When I see that there is a difference in output.js but exec.js still runs, I can review the changes confident that there isn't some bug lurking inside. In case exec != input I need to stop and think if the difference in output is due to the transform changing or to the non relevant.